weave-js/index.html

```diff
@@ -50,7 +50,7 @@
     <meta property="og:image:width" content="400" />
     <meta property="og:image:height" content="400" />
     <meta property="og:image:alt" content="Weave Logo" /> -->
-    <script src="/__frontend/env.js"></script>
+    <script src="./__frontend/env.js"></script>
```

Comment:
I think this doesn't work when I'm running the frontend in `yarn dev` on port 3000 with my backend on 9994?

Reply:
Ok, for now I'll just fallback to defaults.  I think we could get this to work if we just made the vite dev server proxy to the backend...

Reply:
Fixed by adding the proxy line